<launch>
    <include file="$(find rto_bringup_sim)/launch/robot.launch" />

    <node pkg="my_package_tut2" type="dontHitWallsNode.py" name="dontHitWallsNode" output="screen">
         <param name="maxVelo" type="double" value="0.6" />
         <param name="minDist" type="double" value="0.3" />
         <param name="maxDist" type="double" value="1.0" />
         <param name="ROBOT" type="string" value="$(env ROBOT)" />
    </node>
</launch>